.github/workflows/twistlock.yml

```diff
@@ -1,48 +0,0 @@
-name: Twistlock
```

Comment:
I would not delete this just yet, there is a new ticket to do that for when we are assured AWS Inspector works.